cs/ccxt/ws/Client.cs

```diff
@@ -75,7 +75,19 @@ public WebSocketClient(string url, string proxy, handleMessageDelegate handleMes
         public Future future(object messageHash2)
         {
             var messageHash = messageHash2.ToString();
-            return (this.futures as ConcurrentDictionary<string, Future>).GetOrAdd (messageHash, (key) =>  new Future());
+            var future = (this.futures as ConcurrentDictionary<string, Future>).GetOrAdd(messageHash, (key) => new Future());
+            if ((this.rejections as ConcurrentDictionary<string, object>).TryRemove(messageHash, out object rejection))
```

Comment:
@pcriadoperez can't we just remove it from the queue like this? ⬆️

Reply:
not sure i understood the comment, you mean when we remove from the messageQueue?

Reply:
I replaced the addAndUpdate call with this one because it was not supported, I'm just asking if this is enough

Reply:
I checked the docs and I believe it is supported, the problem was it was missing an argument, that I thought was optional. I updated in the latest commit, and was passing the build locally.

Reply:
oh ok let s see how the build goes then